python-plugin/src/test/resources/checks/avoidUnoptimizedVectorImages.py

```diff
@@ -0,0 +1,5 @@
+image0 = """<svg xmlns:svg="http://www.w3.org/2000/svg"><g></g></svg>"""
+image1 = """<svg xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"></svg>""" # Noncompliant {{Avoid using unoptimized vector images}}
+image2 = "<svg><!-- Hello world --></svg>" # Noncompliant {{Avoid using unoptimized vector images}}
+image3 = "<svg><g>...</g><g>...</g></svg>" # Noncompliant {{Avoid using unoptimized vector images}}
+image4 = "<svg><metadata></metadata></svg>" # Noncompliant {{Avoid using unoptimized vector images}}
```

Comment:
Is it really a real use case to store SVG code in a python variable? I don't know much about Python front-end frameworks, but in general we store it in separate files, right?

Which makes me wonder if this rule should be implemented in an "xml" plugin rather

Reply:
I agree with you ... I'm not an expert on python, then I don't know if this case is really relevant or not. I trust in conception idea but maybe I shouldn't.

After discussion with one of our Python experts, he said that this is a real use case if we want to dynamise the generation of several SVG images with a dynamic content.